src/controller/net_controller.c

```diff
@@ -793,7 +762,7 @@ int net_controller_tick(controller *ctrl, uint32_t ticks0, ctrl_event **ev) {
                             }
                         }
 
-                        for(size_t i = 18; i < event.packet->dataLength;) {
+                        for(size_t i = 18 + 4; i < event.packet->dataLength;) {
```

Comment:
What is this + 4 ?

Reply:
I added a uint32 field to the header

Reply:
This could probably be the serial reader's position instead.

Reply:
Done.